### ISSUE TITLE
Assignment 2 submission is ready for grading

### ISSUE THREAD

**Issue by khenrich** [VIS-2129-F2020/kristy_henrich-vis#2]
Hi Carole & Alex, my assignment 2 submission is ready for grading. Please find links here for [.rmd](https://github.com/VIS-2129-F2020/kristy_henrich-vis/blob/master/VIS_Assignment%202.Rmd) and [HTML](https://github.com/VIS-2129-F2020/kristy_henrich-vis/blob/master/VIS_Assignment-2.html). Thank you!

-Kristy

**Comment by bauranov** [comment]
Hi Kristy, Please format the link to the HTML file like this: https://vis-2129-f2020.github.io/kristy_henrich-vis/filename, where *filename* is the name of your assignment. Thanks! Here are some comments:

- At the beginning of a code chunk add {r, message=FALSE} to suppress messages in HTML. When you knit your file, go through the HTML and see if some code chunks are displaying the text you don't want (such as the part where you load libraries). If there is such text, remove it with the code I mentioned above.
- Projection of South America with Italy spatial reference: Picasso in the making.
- Sometimes it makes sense to add names of the countries to the map, you can do that with geom_label().

Good work!

**Comment by khenrich** [comment]
Thanks Alex! Quick question: I've tried updating the HTML link in my assignment 3 submission for this week based on the format you just shared, but I continuously get a 404 error. Instead, I've just copied the full link directly into my issue submission for assignment 3.

Am I doing something incorrectly? Here's the link I'm trying for week 3 in the format you shared:
https://vis-2129-f2020.github.io/kristy_henrich-vis/VIS-Assignment-3

But this is the link that works:
https://github.com/VIS-2129-F2020/kristy_henrich-vis/blob/master/VIS-Assignment-3.html
